project/.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+# enable pre-commit.ci at https://pre-commit.ci/
+
+repos:
+  - repo: https://github.com/abravalheri/validate-pyproject
+    rev: v0.15
+    hooks:
+      - id: validate-pyproject
+
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.5.0
+    hooks:
+      - id: ruff
+        #args: [--fix]
```

Comment:
Is there a reason to deactivate ruff fixing? I find it quite useful for ruff to actually fix formatting issues it discovers